libraries/Microsoft.Bot.Builder.AI.Orchestrator/OrchestratorBotComponent.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.Bot.Builder.AI.Orchestrator
     public class OrchestratorBotComponent : BotComponent
     {
         /// <inheritdoc/>
-        public override void ConfigureServices(IServiceCollection services, IConfiguration configuration, ILogger logger)
+        public override void ConfigureServices(IServiceCollection services, IConfiguration configuration)
```

Comment:
From https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-5.0

```
 Writing logs before completion of the DI container setup in the Startup.ConfigureServices method is not supported:

Logger injection into the Startup constructor is not supported.
Logger injection into the Startup.ConfigureServices method signature is not supported
The reason for this restriction is that logging depends on DI and on configuration, which in turns depends on DI. The DI container isn't set up until ConfigureServices finishes.
```

Reply:
the constraint makes sense - you can add a component that depends on logging - but you wouldn't log the adding